distributions/aws/examples/rds-s3/README.md

```diff
@@ -97,40 +97,55 @@ aws s3 mb s3://$S3_BUCKET --region $CLUSTER_REGION
 
 2. Create RDS Instance
 
-Follow this [doc](https://www.kubeflow.org/docs/distributions/aws/customizing-aws/rds/#deploy-amazon-rds-mysql) to set up an AWS RDS instance. Please follow only section called `Deploy Amazon RDS MySQL`. This RDS Instance will be used by Pipelines and Katib.
+Amazon RDS is a managed service that makes it easy to set up, operate, and scale a relational database in the AWS Cloud. It provides cost-efficient, resizable capacity for an industry-standard relational database and manages common database administration tasks. This RDS Instance will be used by Pipelines and Katib.
 
-To use the latest MySQL version (or a different version) in the CFN template, change the `EngineVersion` property of resource `MyDB` to desired version.
+The folloing instructions will show how to setup an RDS instance to be used by the Kubeflow Pipelines and Katib components.
```

Comment:
nit: folloing -> following

Reply:
duplicate line. Already mentioned in previous para